src/services/iiif-parser.test.js

```diff
@@ -350,50 +350,82 @@ describe('iiif-parser', () => {
     });
   });
 
-  describe('parseMetadata()', () => {
-    it('manifest with metadata property returns a list of key, value pairs', () => {
-      const metadata = iiifParser.parseMetadata(lunchroomManifest);
-      expect(metadata.length).toBeGreaterThan(0);
-      expect(metadata[0]).toEqual({ label: "Title", value: "This is the title of the item!" });
+  describe('getMetadata()', () => {
+    let originalLogger;
+    beforeAll(() => {
+      // Mock console.error function
+      originalLogger = console.log;
+      console.log = jest.fn();
+    });
+    afterAll(() => {
+      // Clen up mock
+      console.log = originalLogger;
     });
 
-    it('manifest without metadata property returns []', () => {
-      // Mock console.error function
-      let originalError = console.error;
-      console.error = jest.fn();
-      const metadata = iiifParser.parseMetadata(volleyballManifest);
-      expect(metadata).toEqual([]);
-      expect(console.error).toBeCalledTimes(1);
-      console.error = originalError;
+    describe('reading only manifest-level metadata', () => {
+      it('manifest with metadata returns a list of key, value pairs', () => {
+        const { manifestMetadata, canvasMetadata } = iiifParser.getMetadata(lunchroomManifest, false);
+        expect(manifestMetadata.length).toBeGreaterThan(0);
+        expect(canvasMetadata.length).toEqual(0);
+        expect(manifestMetadata[0]).toEqual({ label: "Title", value: "This is the title of the item!" });
+      });
+
+      it('manifest without metadata returns []', () => {
+        const { manifestMetadata, canvasMetadata } = iiifParser.getMetadata(volleyballManifest, false);
+        expect(manifestMetadata).toEqual([]);
+        expect(canvasMetadata.length).toEqual(0);
+        expect(console.log).toBeCalledTimes(1);
+      });
+    });
+
+    describe('reading only canvas-level metadata', () => {
```

Comment:
This test is for reading canvas level metadata but the manifest level also gets read, right?
```suggestion
    describe('reading canvas-level metadata', () => {
```